src/installation/index.md

```diff
@@ -26,11 +26,13 @@ Make sure you have [Rust][rust-lang-org] installed. If not, see the instructions
 
 See also [alternative installation methods][rust-alt-installation].
 
-> **Note**: If you run Windows on your host machine, make sure you have installed one of the ABIs listed below. For more details, see the [Windows][rustup-book-windows] chapter in The rustup book.
+> **Warning**: When using Windows, make sure you have installed one of the ABIs listed below. For more details, see the [Windows][rustup-book-windows] chapter in The rustup book.
```

Comment:
> Also, it seems like making Windows and Unix bold will improve the readability. If this seems logical, this can be added to the style guide.

It might make it easier to spot but other Rust book do not use bold for this.